### DISCUSSION TITLE
Insert current timestamp as YYYY/MM/DD

### DISCUSSION THREAD

**Discussion by danielo515** [johnlindquist/kit#1116] [Share]
This snippet is basic, and stupid, but you will be happy to have it around when you need it.
Rather than manually input the current today date, you just type `!tday` and you get it.

[Open today-timestamp in Script Kit](https://scriptkit.com/api/new?name=today-timestamp&url=https://gist.githubusercontent.com/danielo515/9356f1af0b642dd23f6cdc188d73d7be/raw/6f8ca2cf7a666762c533bb47773403f210c55f49/today-timestamp.ts")

```ts
// Name: today-timestamp
// Description: inserts the today date (not including time) formatted as YYYY/MM/DD
// Snippet: !tday

import '@johnlindquist/kit';

function twoDigits(number: number): string {
	return number.toString().padStart(2, '0');
}

await hide();
const today = new Date();
// Format date to YYYY/MM/DD format
const formatted = `${today.getFullYear()}/${twoDigits(today.getMonth() + 1 )}/${twoDigits(today.getDate())}`;
await keyboard.type(formatted);

```

**Comment by SimplGy** [comment]
Gotta remember to add `1` to the formatted JS month -- February is 2, not 1 😁

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth

**Reply by danielo515** [reply]
You're absolutely right. I tend to forget that. Just fixed it, thanks
